### ISSUE TITLE
add 

### ISSUE THREAD

**Issue by emninger** [IgnorantGuru/spacefm#660]
Is it possible to add the following features (whcih pcmanfm has) to spacefm:

 - `pcmanfm menu://applications`
 - `pcmanfm computer:///`
 - `pcmanfm Trash:///`
??
It's awkwarded to add them to spacefm. For menu://applications the only way i found is a shortcut using xdotool .... But probably i'm missing something ...

**Comment by stewie** [comment]
These proposed additions would be exposed as context menu items within the spacefm desktop menu?

**Comment by emninger** [comment]
Not really. For example, devices you can call within spacefm activatin Devices > Show Devices but i do not see any way how to call it by a spacefm + option call. Like it is possible with pcmanfm: 
`pcmanfm computer:///`starts pcmanfm with devices in the foreground.

For trash i installed the plugin, but with the ugly consequence that from now on, any running instance of spacefm is shown in plank (the launcher bar) as an instance of trash, not spacefm.

**Comment by emninger** [comment]
Not really. For example, devices you can call within spacefm activatin Devices > Show Devices but i do not see any way how to call it by a spacefm + option call. Like it is possible with pcmanfm: 
`pcmanfm computer:///`starts pcmanfm with devices in the foreground.

For trash i installed the plugin, but with the ugly consequence that from now on, any running instance of spacefm is shown in plank (the launcher bar) as an instance of trash, not spacefm.

**Comment by emninger** [comment]
Not really. For example, devices you can call within spacefm activatin Devices > Show Devices but i do not see any way how to call it by a spacefm + option call. Like it is possible with pcmanfm: 
`pcmanfm computer:///`starts pcmanfm with devices in the foreground.

For trash i installed the plugin, but with the ugly consequence that from now on, any running instance of spacefm is shown in plank (the launcher bar) as an instance of trash, not spacefm.
